src/custom/constants/index.ts

```diff
@@ -38,3 +38,5 @@ export const BUY_ETHER_TOKEN: { [chainId in ChainId]: Token } = {
 }
 
 export const ORDER_ID_SHORT_LENGTH = 8
+
+export const INPUT_OUTPUT_EXPLANATION = 'Only executed swaps incur fees.'
```

Comment:
nit: This feels like overkill in my eyes. I would favour simplicity over dryness in this case

Reply:
okok